taxcalc/current_law_policy.json

```diff
@@ -1100,10 +1100,10 @@
         "value":     [0.05]        
     },
     "_CTC_additional":{
-        "long_name": "Partially refundable child tax credit",
+        "long_name": "Additional additional child tax credit (Lee-Rubio)",
```

Comment:
Do you think this should be "Additional additional child tax credit amount (Lee-Rubio)?", adding the 'amount'?